README.md

```diff
@@ -6,6 +6,8 @@ node runtime.
 
 [![Build Status](https://travis-ci.org/nodejs/citgm.svg?branch=master)](https://travis-ci.org/nodejs/citgm)
 
+The deployed version of this, used by the Node.js Core team and hosted on multiple platforms, can be found [on our Jenkins CI](https://ci.nodejs.org/view/Node.js-citgm/job/thealphanerd-smoker/).
```

Comment:
I don't know if `deployed` is the best term. We have a CI job set up to run this... but CITGM itself is just a node module, not a server.

Thoughts?